Yoast/wordpress-seo
# Make sure you stay on the same tab when saving.

## Summary

This PR can be summarized in the following changelog entry:

* Make sure you stay on the same tab when saving.

## Relevant technical choices:

* Fix by replacing `#top%23` just like `#top#`. 

## Test instructions

This PR can be tested by following these steps:

* Go to a tab, save changes, see you don't get returned to that tab.
* Pull this PR, do the same, see that you _do_ return to that tab.

Tested this in Safari myself, particularly.

## Quality assurance


Fixes #8813
